source/_patterns/01-atoms/button/_button-dropdown-split.twig

```diff
@@ -0,0 +1,18 @@
+{#
+  Split Button Dropdown!
+#}
+
+  {% include '@atoms/button/_button.twig' with {
+    button_color: dropdown_color,
+    button_text: dropdown_text,
+  } %}
+
+  {% embed '@atoms/button/_button.twig' with {
+    other_classes: 'dropdown-toggle dropdown-toggle-split',
+    other_attributes: 'data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"',
+    button_color: dropdown_color,
+  }%}
+    {% block right %}
+      <span class="sr-only">Toggle Dropdown</span>
```

Comment:
I support using the `sr-only` class, but we should have a chat about this vs. `visually-hidden` that's used within Drupal. The Bootstrap 3.3.7 Drupal theme sticks with `sr-only`, but this is a choice I think we'll have to make for clarity on this project.